package main

// import "fmt"

// func recoverExample() {
// 	defer func() {
// 		if err := recover(); err != nil {
// 			fmt.Println("Recovered from panic:", err)
// 		}
// 	}()
// 	panic("Oops! Something went wrong!")
// }

// func main() {

// 	recoverExample()
// 	fmt.Println("Program continues to execute.")
// }
